### PR TITLE
Readd total push notification health apart of android and ios push notification health

### PR DESCRIPTION
**Pull request by larkox**
#### Summary
When I added notification health for android and iOS separately, old servers were only able to see errors, since they were not yet populating the data with the android / ios labels.

We add the total one so it can be seen along with the android and iOS ones.

#### Ticket Link
NONE

### CODE REVIEW

File: grafana/mattermost-notification-metrics.json [hunk 1]
```diff
@@ -1030,7 +1159,42 @@
           },
           "mappings": []
         },
-        "overrides": []
+        "overrides": [
```
Comment (devinbinnie) [inline grafana/mattermost-notification-metrics.json]: Remove this

File: grafana/mattermost-notification-metrics.json [hunk 2]
```diff
@@ -1175,7 +1339,37 @@
...
```
Comment (devinbinnie) [inline grafana/mattermost-notification-metrics.json]: And this

File: grafana/mattermost-notification-metrics.json [hunk 3]
```diff
@@ -1371,7 +1564,7 @@
   "timepicker": {},
   "timezone": "",
   "title": "Mattermost Notification Health",
-  "uid": "ab852b98-e754-4e9f-9791-6296e9fd127c",
-  "version": 4,
+  "uid": "b03eca8a-6ee3-443e-b16c-7b03f2eb64e8",
```
Comment (devinbinnie) [inline grafana/mattermost-notification-metrics.json]: Revert these, but you can probably still increment by one. Definitely leave the UID the same

**Review by devinbinnie — APPROVED** [review]

File: grafana/mattermost-notification-metrics.json [hunk 4]
```diff
@@ -398,7 +522,7 @@
           },
           "disableTextWrap": false,
           "editorMode": "builder",
-          "expr": "sum(mattermost_notifications_total{type=\"push\", platform=\"ios\"}) OR vector(0)",
+          "expr": "sum(mattermost_notifications_total{type=\"push\" platform=\"ios\"}) OR vector(0)",
```
Comment (agnivade) [inline grafana/mattermost-notification-metrics.json]: Are you 100% sure you don't need a comma? AFAIK a comma is needed.

Comment (larkox) [reply grafana/mattermost-notification-metrics.json]: Probably I messed that up removing namespaces. Will fix :sweat_smile:

**Review by agnivade — APPROVED** [review]

**Pull request merged**